templates/contactDetails.html

```diff
@@ -24,8 +24,10 @@ <h2>
 				</div>
 			</div>
 			<div id="details-actions">
+				<button ng-show="!ctrl.starred" ng-click="ctrl.starContact()" class="icon-star" title="{{ctrl.t.favour}}"></button>
```

Comment:
two buttons? why not change the class inside.

Reply:
Didn't thought about that! Good point